YipYapTimeAPI/README.md

```diff
@@ -36,7 +36,7 @@ Features
 ### Prerequisites
 Before you begin, ensure you have the following installed on your system:
 
-- Java 11 or higher
+- Java 21 or higher
```

Comment:
change to Java 17, dont change to java 21 until JVM and compiler testing is done